packages/vetrina/src/Vetrina/Purchase/NewPurchase/Order.purs

```diff
@@ -131,11 +131,11 @@ mkPurchase props logger window askAccount validForm affUser = do
         let pError = props.purchaseError (hush eitherUser)
         case err of
           EmailInUse email ->
-            pError (Just $ ExistingAccount email) validForm.productSelection NewPurchase
+            pError (Just $ ExistingAccount { email: email, invalidPassword: false }) validForm.productSelection NewPurchase
```

Comment:
This could use short hand format `{email, invalidPassword: true}`. Same for the `ExistingAccount` use below.

Reply:
Fixed!